src/azsdkdemoprocessor/Dockerrun.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker run --env-file=./../../.env jongio/azsdkdemoprocessor:latest
```

Comment:
You may want to generalize the tag prefix rather than using your alias.

Reply:
The whole thing is under jongio GH, so I'm okay with this for now.  If this develops into something else, sure.